src/libguac/guacamole/user.h

```diff
@@ -509,6 +509,27 @@ struct guac_user {
      */
     guac_user_argv_handler* argv_handler;
 
+    /**
+     * Handler for mouse events sent by the Gaucamole web-client.
```

Comment:
mouse?

Reply:
Whoops. Fixed this one via rebase, too.

🧀 🐁